flint/integ-test/src/test/scala/org/opensearch/flint/spark/FlintSparkSqlSuite.scala

```diff
@@ -48,13 +48,29 @@ class FlintSparkSqlSuite extends QueryTest with FlintSuite with OpenSearchSuite
            |""".stripMargin)
   }
 
-  test("drop skipping index") {
+  protected override def beforeEach(): Unit = {
+    super.beforeEach()
+
     flint
       .skippingIndex()
       .onTable(testTable)
```

Comment:
if these unit tests are run in parallel, we should consider adding a random tag to the table name so that all unit tests run against a different table/index.

Reply:
I renamed it in https://github.com/opensearch-project/sql/pull/1785/commits/807a1be4ed2210d373dd299a4deb18d2e9368b07. Thanks!